Apps/CommonData/src/ViewModels/PrimitiveRequestResult.cs

```diff
@@ -15,52 +15,13 @@
 //-------------------------------------------------------------------------
 namespace HealthGateway.Common.Data.ViewModels
 {
-    using System.Text.Json.Serialization;
-    using HealthGateway.Common.Data.Constants;
-
     /// <summary>
     /// Class that represents the result of a request with a primitive values.
     /// Contains members for handling pagination and error resolution.
     /// Note: Will not be necessary on the next release of c#.
     /// </summary>
     /// <typeparam name="T">The payload type.</typeparam>
-    public class PrimitiveRequestResult<T>
-        where T : struct
+    public class PrimitiveRequestResult<T> : RequestResult<T>
     {
-        /// <summary>
```

Comment:
This class is now empty.  Is it still being referenced?  Do we still need it?  Can we delete it?

Reply:
It is referenced and it would take more refactoring to be able to remove it